classic-commerce.php

```diff
@@ -150,6 +150,11 @@ function cc_wc_activate_attempted_notice() {
 		define( 'WC_PLUGIN_FILE', __FILE__ );
 	}
 
+	// Define WC_PLUGIN_DIR.
+	if ( ! defined( 'WC_PLUGIN_DIR' ) ) {
+		define( 'WC_PLUGIN_DIR', plugin_dir_path( WC_PLUGIN_FILE ) );
+	}
+
```

Comment:
If we use `WC_ABSPATH` do we still need this addition?

Reply:
Well spotted. That will be gone very soon.

Reply:
Thanks my last change would be on `includes/class-woocommerce.php` line 24. Doesn't that need a bump as well?

Reply:
That's been bumped Laurence. Everything that needs to be bumped, has been bumped.